spark_ec2.py

```diff
@@ -248,7 +251,7 @@ def parse_args():
              "(default: %default).")
     parser.add_option(
         "--hadoop-major-version", default="yarn",
-        help="Major version of Hadoop. Valid options are 1 (Hadoop 1.0.4), 2 (CDH 4.2.0), yarn " +
+        help="Major version of Hadoop. Valid options are 1 (Hadoop 1.0.4), 2 (CDH 4.2.0), yarn, hadoop2.7.7 (Hadoop 2.7.7)" +
```

Comment:
--hadoop-major-version needs to be hadoop-2.7.7  because the filename is named as hadoop-2.7.7.tar.gz.